web/eXoResources/src/main/webapp/javascript/eXo/webui/UIRightClickPopupMenu.js

```diff
@@ -251,10 +251,15 @@
 	      break;
 	    default:
 	      // if it isn't fit to be showed down BUT is fit to to be showed up
-	      if ((eXo.core.Mouse.mouseyInClient + ctxMenuContainer.offsetHeight) > browserHeight
-	          && (intTop > ctxMenuContainer.offsetHeight)) {
-	        intTop -= ctxMenuContainer.offsetHeight;
+	      if ((eXo.core.Mouse.mouseyInClient + intTop) > browserHeight )
+	         {
+	        if(document.getElementById('NavigationNodePopupMenu'))document.getElementById('NavigationNodePopupMenu').classList.add("dropup");
+	        if(document.getElementById('ECMContextMenu'))document.getElementById('ECMContextMenu').classList.add("dropup");

```

Comment:
We can not use a class from ECMS module here, the fix should work without attaching it to the module context.

Reply:
Well understood, thank you.

Reply:
> the fix should work without attaching it to the module context.

so even when using this "NavigationNodePopupMenu" isn't allowed, isn't it?

Reply:
well understood, thank you.